### ISSUE TITLE
Add explanations for the Mila dashboard main page

### ISSUE THREAD

**Issue by satyaog** [mila-iqia/mila-docs#52]
![Screen Shot 2021-08-07 at 3 51 55 PM](https://user-images.githubusercontent.com/5014902/128612366-84f8bd1a-095d-4446-859f-d623e274ba7e.png)
Can users access this dashboard? What is this dashboard? Why is there no explanation for it?

_Originally posted by @tesfaldet in https://github.com/mila-iqia/mila-docs/issues/46#issuecomment-894700074_

**Comment by btravouillon** [comment]
Yes they can. It is now documented at https://docs.mila.quebec/Information.html#monitoring, see #26.

On the login page there is a "Sign-in with Google" button. It seems self-explanatory to me that one need to use their Google account @mila.quebec. (I might be wrong though).
